src/sage/categories/enumerated_sets.py

```diff
@@ -494,7 +495,7 @@ def tuple(self):
 
             EXAMPLES::
 
-                sage: (GF(3)^2).tuple()
+                sage: (GF(3)^2).tuple()                                                 # optional - sage.libs.pari
                 ((0, 0), (1, 0), (2, 0), (0, 1), (1, 1), (2, 1), (0, 2), (1, 2), (2, 2))
```

Comment:
Maybe split this result so it fits in 80 columns? There are a few more below, some even a bit longer.

Reply:
Column 80 plays no role in my undocumented style guide. I let the output run to column 86 regularly, and sporadically to column 96. (After that, horizontal scrolling is necessary.)